platform-hub-api/db/structure.sql

```diff
@@ -0,0 +1,256 @@
+--
```

Comment:
I am using migrations; this is auto generated by Rails. See http://guides.rubyonrails.org/active_record_migrations.html#schema-dumping-and-you